docs/meta-box.md

```diff
@@ -56,15 +48,10 @@ between the `MetaBoxIframe` and the Redux Store. *If no meta boxes are active,
 nothing happens. This will be the default behavior, as all core meta boxes have
 been stripped.*
 
-#### MetaBoxIframe Component
```

Comment:
There are still two lingering references to `MetaBoxIframe` components in this README (in section "Redux and React Meta Box Management")